src/background.js

```diff
@@ -43,6 +50,15 @@ chrome.webNavigation.onBeforeNavigate.addListener(function(data) {
                 } else {
                   return chrome.tabs.update(data.tabId, { url: data.url });
                 }
+              }); //
+            } else if (url.href.startsWith("https://explorer.zksync.io/search")) {
```

Comment:
@nidz-the-fact Are you sure this will work? Can you go to `https://explorer.zksync.io/search` and enter "techie.zksoul" and see if it redirects you to the domain page?